TESTING.md

```diff
@@ -44,38 +44,34 @@ __Your session will periodically expire__, so you'll need to run this command to
 
 Remote tests can be run against real devices provided by BrowserStack. In order to run these tests, you need:
 
-A BrowserStack App Automate Username: `BROWSER_STACK_USERNAME`
-A BrowserStack App Automate Access Key: `BROWSER_STACK_ACCESS_KEY`
+A BrowserStack App Automate Username: `MAZE_DEVICE_FARM_USERNAME`
+A BrowserStack App Automate Access Key: `MAZE_DEVICE_FARM_ACCESS_KEY`
 A local docker and docker-compose installation.
 
 ### Instrumentation tests
 
 Ensure that the following environment variables are set:
 
-* `BROWSER_STACK_USERNAME`: The BrowserStack App Automate Username
-* `BROWSER_STACK_ACCESS_KEY`: The BrowserStack App Automate Access Key
+* `MAZE_DEVICE_FARM_USERNAME`: The BrowserStack App Automate Username
+* `MAZE_DEVICE_FARM_ACCESS_KEY`: The BrowserStack App Automate Access Key
 
 Run `make remote-test`
 
 ### End-to-end tests
 
 Ensure that the following environment variables are set:
-* `BROWSER_STACK_USERNAME`: Your BrowserStack App Automate Username
-* `BROWSER_STACK_ACCESS_KEY`: You BrowserStack App Automate Access Key
+* `MAZE_DEVICE_FARM_USERNAME`: Your BrowserStack App Automate Username
```

Comment:
Will this break things for existing users or as is there backwards compatibility?

Reply:
The `--username` and `--access-key` command line options are still available, so this won't break people still using the old approach.  They will only need to change the environment variables they use if they want to take advantage of MazeRunner's ability to read config from the environment.  It's maybe worth noting that `BROWSER_STACK_USERNAME` was never a feature of MazeRunner itself - it was just very commonly used in the `env.rb` files - closely associated with MazeRunner, of course, but not a part of the tool itself.